documentation/wiki/Providing-Binary-Logs.md

```diff
@@ -12,3 +12,6 @@ You can create a binary log by passing the `-bl` parameter to MSBuild.  You can
 
 ## Capturing Binary Logs Through Visual Studio
 See [this guide](https://github.com/dotnet/project-system-tools) in the Project System Tools repo for capturing binlogs through Visual Studio.
+
+## Viewing Binary Logs Online
```

Comment:
I'd move this up and combine it with "You can explore the contents.../)."

Reply:
Good idea 👍